### PR TITLE
Update seed

### PR DESCRIPTION
**Pull request by amhursh**
Related Issue: #139 

Summary: Fixed user validations and seed file

Author(s): amhursh

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -2,7 +2,9 @@ class User < ApplicationRecord
   has_secure_password
   has_many :orders
   validates :username, uniqueness: true, presence: true
-  validates :password_digest, :full_name, :address, presence: true
+  validates :password, presence: :true
```
Comment (limsammy) [inline app/models/user.rb:5]: Weird how this fixed the problem.

File: db/seeds.rb [hunk 2]
```diff
@@ -9,7 +9,7 @@
 Category.create!(title: "Drums")
 
 categories = Category.all
-user = User.create(username: 'sam', password: 'sam', role: 0)
+user = User.create(username: 'sam', password: 'sam', full_name: 'sam lim', address: 'sams incubator', role: 0)
```
Comment (limsammy) [inline db/seeds.rb:12]: Good catch

File: app/models/user.rb [hunk 3]
```diff
@@ -2,7 +2,9 @@ class User < ApplicationRecord
...
+  validates :address, presence: :true
+  validates :full_name, presence: :true
```
Comment (randy-darsh) [inline app/models/user.rb:7]: nice catch on the validations

**Review by limsammy — APPROVED** [review]
Looks good

File: db/seeds.rb [hunk 4]
```diff
@@ -23,4 +23,4 @@
   puts "Created #{item.title} with category #{item.category.title}"
 end
 
-User.create(username: 'admin', password: 'admin', role: 'admin')
\ No newline at end of file
+User.create(username: 'admin', password: 'admin', full_name: 'admin admin', address: 'admin street, admintown', role: 'admin')
```
Comment (randy-darsh) [inline db/seeds.rb:26]: looks good

**Comment by randy-darsh** [comment]
looks good. MERGING... ENGAGE!

**Pull request merged**